### ISSUE TITLE
api: Implement ListBucketPolicies 

### ISSUE THREAD

**Issue by harshavardhana** [minio/minio-py#409]

**Comment by indradhanush** [comment]
@harshavardhana I could take a dig at this.

**Comment by harshavardhana** [comment]
Thanks @indradhanush

**Comment by MrRedAmber** [comment]
@indradhanush checking in, looking to pickup this issue.

**Comment by indradhanush** [comment]
@MrRedAmber I haven't been able to get to this. :(
You can go ahead if you want. 👍

**Comment by harshavardhana** [comment]
// cc @MrRedAmber is working on this and has a patch ready for us as well.

**Comment by deekoder** [comment]
Waiting on @MrRedAmber  to respond on the open PR.

**Comment by kannappanr** [comment]
Closing this issue as a duplicate of #611
